site/docs/foundations/typography.mdx

```diff
@@ -166,135 +41,94 @@ Weight refers to the relative thickness of a font’s stroke. Each style include
-
 </DesignCard>
 
 </DesignCards>
 
-## Alignment
-
-Type alignment controls how text aligns in the space. There are four types of alignment: left, right, centered and justified.
-
-### Left-aligned
-
-Text aligned to the left is the default configuration for left-to-right languages, such as English.
+## Type hierarchy
 
-### Right-aligned
+Type hierarchy arranges text elements to guide the reader's attention and convey content structure. By varying font sizes, weights, and styles, it creates a clear visual order for easier navigation and understanding.
 
-Right align text for right-to-left languages, such as Arabic or Hebrew. It should also be used for Numeric values (including dates, times, and currency) for comparisons within a data set, for instance within a data grid column. If a numeric value is displayed in isolation, and therefore not for comparison, it should be left aligned to fit the content.
-
-<ExampleContainer type="positive">
-  <Image
-    src="/img/foundations/typography-rightalign-annotations.svg"
-    alt="Example of right aligned text"
-  />
-  Right-align short amounts of text, such as a supporting detail, side note or
-  annotation.
-</ExampleContainer>
-
-<ExampleContainer type="positive">
-  <Image
-    src="/img/foundations/typography-rightalign-numbers.svg"
-    alt="Example of right aligned text"
-  />
-  Right-align numerical values and corresponding headers within data grid
-  columns.
-</ExampleContainer>
+#### Display
 
-### Center-aligned
+Display styles are primarily for representing key metrics and mastheads.
 
-You should only center-align short sentences or blocks of text included as typographic elements, such as a pull quote or key metric. Avoid center-aligning large blocks of text or headings. Longer text forces readers to make additional eye movements on both sides of the text, resulting in unnecessary eye fatigue and lower reading accuracy.
+<Image
+  src="/img/foundations/typography-style-display.svg"
+  alt="Graphical representation of Display styles"
+/>
 
-### Justified text
+#### Headings
 
-The use of justified text is not encouraged, but if required for a more formal aesthetic or to align for print materials, consider enabling hyphenation to balance negative space and avoid large gaps between words.
+Headings provide varying levels of typographic hierarchy.
 
-## Line length
+<Image
+  src="/img/foundations/typography-style-headings.svg"
+  alt="Graphical representation of Heading styles"
+/>
 
-When including paragraphs of text within a design, the layout should accommodate the intended reader. Shorter line lengths are typically more comfortable for most readers and scenarios. With longer line lengths, a reader's eyes have to travel further from the end of one line to the start of the next. This can make it hard to track progress and can cause fatigue (Baymard Institute, 2022).
+#### Body
```

Comment:
Display & heading has image, Body/label/help doesn't have them?

Reply:
@amritadesmet mentioned that she wanted to focus on the content first and the images would be added when she looks at the design of the page.